msg-socket/src/rep/driver.rs

```diff
@@ -25,15 +25,14 @@ pub(crate) struct PeerState<T: AsyncRead + AsyncWrite> {
     addr: SocketAddr,
     egress_queue: VecDeque<reqrep::Message>,
     state: Arc<SocketState>,
-    flush_interval: Option<tokio::time::Interval>,
-    should_flush: bool,
```

Comment:
We should keep this and check it in the poll loop so we don't call `poll_flush` even without data to write

Reply:
fixed